css/main.css

```diff
@@ -7,6 +12,7 @@ body {
     -o-background-size: cover;
     background-size: cover;
   font-family: 'Quattrocento', sans-serif;
+  margin-bottom: 0 0 25px;
```

Comment:
This should just be 'margin-bottom: 25px' or 'margin: 0 0 25px'. 
You should also consider increasing this number to something like 75px so there is more space between the content and footer.